ground/gcs/src/plugins/config/configoutputwidget.cpp

```diff
@@ -102,18 +102,6 @@ ConfigOutputWidget::ConfigOutputWidget(QWidget *parent)
     connect(m_config->calibrateESC, &QAbstractButton::clicked, this,
             &ConfigOutputWidget::startESCCalibration);
 
-    // Track the ActuatorSettings object
-    addUAVObject("ActuatorSettings");
-
-    // Associate the buttons with their UAVO fields
-    addWidget(m_config->cb_outputRate6);
```

Comment:
These cause a segfault when ConfigTaskWidget adds the relations itself... and don't actually do anything AFAICT.